scripts/generate_road_colours.py

```diff
@@ -32,7 +32,10 @@ def rgb_error(self):
 
 def load_settings():
     """Read the settings from YAML."""
-    return yaml.load(open('road-colors.yaml', 'r'))
+    with open('road-colors.yaml', 'r') as fd:
+        y=yaml.load(fd, Loader=yaml.SafeLoader)
```

Comment:
Nitpick: could you add spaces around `=` like in the rest of the file?

Reply:
`autopep8` should catch such formatting, and fix most of them

Reply:
If you want to enforce pep8 (any codestyle), check should be added to CI (.travis.yml)

Reply:
But of course this should not be part of this PR.